src/main/resources/lang/en_US.json

```
@@ -220,7 +220,9 @@
     "showFeatureNamesOnHover": "Show Feature Names on Hover",
     "resetSalvagedEssencesAfterLeavingMenu": "Reset Salvaged Essence Counter After Closing Menu",
     "changeDungeonMapZoomWithKeyboard": "Adjust zoom with +/- keys",
-    "showProfileTypeInChat": "Show Profile Type In Chat"
+    "showProfileTypeInChat": "Show Profile Type In Chat",
+    "crimsonArmorAbilityStacks": "Display Crimson-Armor Ability Stacks",
```

This is not even english
```suggestion
    "crimsonArmorAbilityStacks": "Crimson Armor Stacks",
```

---

bro who are you

---

how tf is that not english, also

  - we dont have multiple stacks of the armor, we have multiple stacks of the ability of said armor
  - a lot of sba features are prefixed with "Display", "Hide", etc.
  - What do you meeean its not english, all you did is remove a dash, did that singular dash make it inrecognizable to you?? are you sure you are actually human

---

if youre gonna have an aneurysm over a dash, at least dont be so fucking condescending holy shit

---

The stacks are from the crimson armor, adding ability is just unnecessary as that can be easily deducted from the context. Also yes, dashes are not used like that in English. And no, feature names don’t need a verb before them, I think it’s perfectly recognizable this way